.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,47 @@
+name: Uni-Dock Benchmark
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Docker image used"
+        required: false
+        default: "latest"
+jobs:
+  unidock_benchmark:
+    runs-on: nvidia
+    container:
+      image: dptechnology/unidock:${{ github.event.inputs.tag }}
+      options: --gpus all
+    steps:
+      - name: Checkout test suites
+        uses: actions/checkout@v4
+        with:
+          repository: dptech-corp/Uni-Dock-Benchmarks
```

Comment:
Since there are many small files in the Uni-Dock-Benchmark repo (test data in molecular_docking) and the file size is large (test data in virtual_screening, like GBA), it takes a long time to pull the data each time. If it is on a fixed runner, can we first check if there is this batch of data, and then decide whether to pull it or not? Or create a base image with a test dataset. @caic99 @YNYuan

Reply:
Hi @zhengh96 ,

>  it takes a long time to pull the data each time. 

This is false. It takes seconds to checkout the Benchmarks repo, and <1min to download data from external source.